project/scalafmt.conf

```diff
@@ -1,3 +1,4 @@
+maxColumn = 120
```

Comment:
.............

Reply:
don't get me started. You see what a mess it's making of by-name?

LOOK WHAT YOU MADE ME DO!